index.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 var execall = require('execall');
-var re = /^\s*[^$]\s*(?:-([a-z-]), )?--([a-z-]+) +(.*)$/igm;
+var re = /^\s*[^$]\s*(?:-([a-z-]),\s+)?--([a-z-]+) +(.*)$/igm;
```

Comment:
I think `\s` always matches newlines:

> `\s` - Matches a single white space character, including space, tab, form feed, line feed and other Unicode spaces. Equivalent to [ \f\n\r\t\v​\u00a0\u1680​\u180e\u2000​-\u200a​\u2028\u2029\u202f\u205f​\u3000\ufeff]. - MDN

-

` +` works, or you could use the space escape explicitly `\u0020+`